next/overview/README.md

```diff
@@ -0,0 +1,39 @@
+
+# 3D Tiles Next Overview
```

Comment:
Now that #546 is merged, could you pull in `main` and add a link to the overview page (and perhaps preview image) to the `next` page?